dimension/store.go

```diff
@@ -90,13 +98,13 @@ func (store *Store) processRequest(r *http.Request, instanceID string) ([]byte,
 	}
 	switch response.StatusCode {
 	case http.StatusNotFound:
-		return nil, errors.New("Failed to find instanceId : " + instanceID)
+		return nil, ErrInstanceNotFound
```

Comment:
Is there not still value in passing the instanceID out?

Reply:
The instance ID gets attached to any errors at a higher level 👍